frontend/packages/console-shared/src/components/status/PopoverStatus.tsx

```diff
@@ -37,7 +36,7 @@ type PopoverStatusProps = {
   title?: string;
   hideHeader?: boolean;
   isVisible?: boolean;
-  shouldClose?: (tip: TippyInstance) => void;
+  shouldClose?: (hideFnc: any) => void;
```

Comment:
Can you improve the types here?

nit: I'd spell it out to make it more clear: `hideFunction` (I'm assuming fnc means function)

Reply:
Fixed, changed to `hideFunction`.  Can you pls elaborate on how to improve types?  Using what PF-React has for [Popover](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/Popover/Popover.tsx#L93)

Reply:
Looking at the code it seems PF didn't update their types.

`shouldClose?: (hideFnc: () => void) => void;`